setup.cfg

```
@@ -0,0 +1,39 @@
+[metadata]
+name=pydcm2niix
```

would prefer:

```suggestion
name=dcm2niix
```

but was getting errors:

> The name 'dcm2niix' isn't allowed. See https://pypi.org/help/#project-name for more information.

---

> @jstutters are you OK with this pull request? I notice you have a repository named [pydcm2niix](https://github.com/jstutters/pydcm2niix).

Interesting. Ideally this PR should use simply `dcm2niix` anyway. I can raise a request at https://www.python.org/dev/peps/pep-0541/#how-to-request-a-name-transfer if you want @neurolabusc?